speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DataSelectionMenu.cpp

```diff
@@ -78,6 +78,15 @@ void InitializeDataSelectionSettings(void* p_param)
     InitializeDataSelectionButtons();
 }
 
+void InitializeDefaultDataSettings()
+{
+    m_dataToStore.EnvironmentData = GfuiCheckboxIsChecked(s_scrHandle, m_dataToStoreControl[0]);
+    m_dataToStore.EnvironmentData = GfuiCheckboxIsChecked(s_scrHandle, m_dataToStoreControl[1]);
+    m_dataToStore.EnvironmentData = GfuiCheckboxIsChecked(s_scrHandle, m_dataToStoreControl[2]);
+    m_dataToStore.EnvironmentData = GfuiCheckboxIsChecked(s_scrHandle, m_dataToStoreControl[3]);
+    m_dataToStore.EnvironmentData = GfuiCheckboxIsChecked(s_scrHandle, m_dataToStoreControl[4]);
```

Comment:
I don't think you meant to set the EnvironmentData 5 times :P

Reply:
I feel stupid now